followthemoney/statement/entity.py

```diff
@@ -163,6 +165,7 @@ def add_statement(self, stmt: Statement) -> None:
                 else:
                     self.last_change = max(self.last_change, stmt.first_seen)
         else:
+            self._caption = None
```

Comment:
I don't think crazy important: does this want a `if stmt.prop in self.schema.caption`?